common/mocks/ClusterMetadata.go

```diff
@@ -0,0 +1,114 @@
+// Copyright (c) 2017 Uber Technologies, Inc.
```

Comment:
since metadataTestBase.go is provided, this mock does not do much, is it?

Reply:
I'm using it in some unit test to assert IsGlobalDomain API.